packages/safe-core-sdk-types/src/contracts/GnosisSafeContract.ts

```diff
@@ -22,6 +22,6 @@ export interface GnosisSafeContract {
     safeTransaction: SafeTransaction,
     options?: TransactionOptions
   ): Promise<TransactionResult>
-  encode(methodName: any, params: any): string
+  encode(methodName: string, params: any): string
   estimateGas(methodName: string, params: any[], options: TransactionOptions): Promise<number>
```

Comment:
And these `params`?

Reply:
I'm resolving this after discussing it. Please see my comment above for the reasoning.